resources/public/api/conf/6.0/common/errors.yaml

```diff
@@ -12,10 +12,10 @@ components:
 
     formatTaxYear:
       description: |
-        The format of the supplied taxYear value is not valid.
+        The format of the supplied taxYearClaimedFor value is not valid.
       value:
         code: FORMAT_TAX_YEAR
-        message: The taxYear format is invalid
+        message: The provided tax year claimed for is invalid
```

Comment:
Please revert these changes as they are incorrect. Please use [this](https://github.com/hmrc/individual-losses-api/blob/main/resources/public/api/conf/6.0/common/errors.yaml#L20-L25) and update [message](https://github.com/hmrc/individual-losses-api/blob/main/resources/public/api/conf/6.0/common/errors.yaml#L25) to [The provided tax year claimed for is invalid](https://github.com/hmrc/individual-losses-api/blob/main/app/common/errors/LossesApiCommonErrors.scala#L29), thanks. The message `The provided Start date is invalid` is wrong as it isn't a start date and has probably been carried over from previous versions.